src/modules/users/users.controller.ts

```diff
@@ -319,9 +335,9 @@ export class UsersController {
     };
```

Comment:
Please return the updated user here. Additionally, how can I access the user's uploaded photo? The URL format https://{domain}/uploads/avatars/file-****.jpg returns a 404 error.